modules/compliance-anatomy.adoc

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Is the underscore after "compliance_" intentional for the heading IDs in this module? If not, for the main module heading (and several headings below):
s/id="compliance_anatomy_{context}"/id="compliance-anatomy_{context}"/